package.json

```diff
@@ -22,16 +22,16 @@
     "build-production-client": "NODE_ENV=production BABEL_ENV=production yarn build-client",
     "build-production": "yarn distclean && yarn && gulp languages:extract && yarn build-production-client",
     "build-languages": "gulp languages",
-    "docker:up": "docker-compose -f docker/docker-compose.yml up",
+    "docker:up": "docker-compose -f docker/docker-compose.yml up -d",
     "docker:stop": "docker-compose -f docker/docker-compose.yml stop",
     "docker:down": "docker-compose -f docker/docker-compose.yml down",
     "docker:build": "docker-compose -f docker/docker-compose.yml build",
-    "docker:wp": "docker container exec jetpack_wordpress wp --allow-root --path=/var/www/html/",
-    "docker:install": "yarn docker:wp core install --url=localhost --title='Hello World' --admin_user=wordpress --admin_password=wordpress --admin_email=wordpress@example.com --skip-email && yarn docker:wp plugin activate jetpack",
-    "docker:multisite-convert": "yarn docker:wp core multisite-convert && yarn docker:wp config set DOMAIN_CURRENT_SITE 'DOCKER_REQUEST_URL' --raw --type=constant && cp ./docker/htaccess-multi ./docker/wordpress/.htaccess",
-    "docker:sh": "docker exec -it jetpack_wordpress bash",
-    "docker:phpunit": "docker container exec jetpack_wordpress phpunit --configuration=/var/www/html/wp-content/plugins/jetpack/phpunit.xml.dist",
-    "docker:tail": "tail -f docker/logs/php/errors.log",
+    "docker:wp": "docker-compose -f docker/docker-compose.yml run --rm wordpress wp --allow-root --path=/var/www/html/",
```

Comment:
This seems to be making composer attempt to launch containers and remove them instead of running the command on the running container.